tests/endpoints/test-class-data-endpoint.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Class Test_Class_Data_Endpoint.
+ *
+ * @package WP_Irving
+ */
+
+namespace WP_Irving;
+
+use WP_Irving\REST_API;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_Error;
+use WP_UnitTestCase;
+
+/**
+ * Test the data endpoint functionality.
+ *
+ * @group endpoints
+ */
+class Test_Class_Data_Endpoint extends WP_UnitTestCase {
+
+	/**
+	 * Class setup.
+	 */
+	public static function wpSetUpBeforeClass() {
```

Comment:
I think this needs to use the `setUp()` method or else additional tests will not have this filter set due to hooks being reset after each test is run. It's fine for now since there is only one test method.

Reply:
Good call. Thanks.